web/src/popover_menus.js

```diff
@@ -967,6 +967,11 @@ export function initialize() {
                 });
             });
 
+            $(".focus-dropdown").on("focus", (e) => {
```

Comment:
Why do we have a `focus-dropdown` handler inside `onMount`? I basically don't understand this commit.

Reply:
In that case, we should make this event handler specific to this popover and not add global event handler. Use `$popper`